base/applications/network/nslookup/nslookup.c

```diff
@@ -822,6 +826,7 @@ int main( int argc, char* argv[] )
 
     HeapFree( ProcessHeap, 0, pNetInfo );
 
+
```

Comment:
This seems unnecessary.

Reply:
Yes. Sorry for being inconsistent. Also updated in the latest version.